projects/tuist/fixtures/tvos_app_with_uitest/Project.swift

```diff
@@ -0,0 +1,27 @@
+import ProjectDescription
+
+let project = Project(
+    name: "App",
+    targets: [
+        Target(
+            name: "App",
+            platform: .tvOS,
+            product: .framework,
```

Comment:
This is why the test is failing.

```suggestion
            product: .app,
```

Reply:
ahh, it's my mistake. I have corrected it now. Thank you 😊